### PR TITLE
Region-aware capability in marathon

### PR DESCRIPTION
**Pull request by meln1k**
Summary: added region-aware capability and store the default region in a MarathonScheduler

JIRA issues: MARATHON_EE-1663

### CODE REVIEW

File: src/main/scala/mesosphere/marathon/MarathonScheduler.scala [hunk 1]
```diff
@@ -144,9 +148,19 @@ class MarathonScheduler @Inject() (
     }
   }
 
+  protected def updateDefaultRegion(masterInfo: MasterInfo): Unit = {
```
Comment (alenkacz) [inline src/main/scala/mesosphere/marathon/MarathonScheduler.scala]: It is called home region in the design doc so maybe we should stick with that? https://github.com/mesosphere/marathon-design/commit/813c6a1f303b12edb65a0ca1c8eae41c8f6e9d7f#diff-af3ac823b06139f90539055737e0a188

Comment (timcharper) [reply src/main/scala/mesosphere/marathon/MarathonScheduler.scala]: Yeah, let's stick with home region nomenclature; I think this is what they are calling it in other DCOS components

**Comment by timcharper** [comment]
Looks like you also solved https://jira.mesosphere.com/browse/MARATHON_EE-1664 at the same time, @meln1k ?

**Comment by meln1k** [comment]
@timcharper it looks like it :)

**Pull request merged**